modules/linear_boltzmann_solvers/c_discrete_ordinates_curvilinear_solver/sweep_chunks/lbs_curvilinear_sweep_chunk_pwl.h

```diff
@@ -14,7 +14,6 @@ class SweepChunkPWLRZ : public lbs::AAH_SweepChunk
 {
   //  Attributes
 private:
-  const std::vector<lbs::UnitCellMatrices>& secondary_unit_cell_matrices_;
```

Comment:
Just for my own edification, what were these used for before? Or were they used at all?